models/tenant.rb

```diff
@@ -0,0 +1,15 @@
+class Tenant
+  attr_accessor :id, :name, :age, :apartment_id
+  @@ruby_tenants = []
+  def initialize(name, age, apartment_id)
+    @id = @@ruby_tenants.length +1
+    @name = name
+    @age = age
+    @apartment_id = apartment_id
+    @@ruby_tenants << self
+  end
+end
+index=0
+tenants.each do |tenants|
+  tenants = Tenant.new(tenants[index][:address], tenants[index][:monthly_rent], tenants[index][:square_feet])
```

Comment:
Where are you putting each `tenant`?

Also, should give the iteration variable a different name than the collection `.each` is being called on.